projects/core/src/lib/fire-uploader-ref.ts

```diff
@@ -252,7 +252,7 @@ export class FireUploaderRef {
 
         for (let i = 0; i < length; i++) {
           // Check if file type is accepted
-          if (this.config.accept && !fileList[i].type.match(this.config.accept)) {
+          if (this.config.accept && (!fileList[i] || !fileList[i].type.match(this.config.accept))) {
```

Comment:
I thought this might be the `type` from the error `TypeError: Cannot read property 'type' of null`.